### PR TITLE
:construction_worker: Enable daily dependabot updates

### PR DESCRIPTION
**Pull request by hickeyma**
With this PR, [Dependabot](https://dependabot.com/) can automatically scan the dependencies daily as described in `pyproject.toml` and create a PRif there is a newer version of the upstream package available.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily
```
Comment (Nhan-Hoang) [inline .github/dependabot.yml:6]: consider closing the `"`

Comment (hickeyma) [reply .github/dependabot.yml]: Nice catch, thanks. Updated

**Review by joerunde — APPROVED** [review]
lgtm, I double checked and the dependabot docs say `pip` is good with pyproject.toml files

**Review by Nhan-Hoang — APPROVED** [review]
LGTM

**Pull request merged**